### PR TITLE
`solveset_real(abs(f1/f2), x)` for f1,f2 polynomials in `x`

### PR DESCRIPTION
**Pull request by gxyd**
Now

```
>>> solveset_real(abs((x - 1)/(2*x - 3)) - 1, x)    # earlier it raised a `Not-Implemented-Error`
FiniteSet(S(4)/3, 2)
```

### CODE REVIEW

**Comment by gxyd** [comment]
@hargup @aktech  please have a look.

File: sympy/solvers/solveset.py [hunk 1]
```diff
@@ -456,7 +456,7 @@ def solveset_real(f, symbol):
                         result += _solve_radical(equation,
                                                  symbol,
                                                  solveset_real)
-                    elif equation.has(Abs):
+                    elif equation.has(Abs) and not equation.as_numer_denom()[1].has(symbol):
```
Comment (hargup) [inline sympy/solvers/solveset.py]: Why does this work?

File: sympy/solvers/tests/test_solveset.py [hunk 2]
```diff
@@ -524,6 +524,7 @@ def test_solve_abs():
         FiniteSet(-1, Rational(1, 3))
 
     assert solveset_real(Abs(x - 7) - 8, x) == FiniteSet(-S(1), S(15))
+    assert solveset_real(Abs((x - 1)/(x - 3)) - 1, x) == FiniteSet(2)
```
Comment (hargup) [inline sympy/solvers/tests/test_solveset.py]: Please add more tests:
* polynomial have degree more than 1.
* `deg(denominator) > deg(numerator)`
* `deg(numerator) > deg(denominator)`

Comment (gxyd) [inline sympy/solvers/tests/test_solveset.py]: test cases added

**Comment by gxyd** [comment]
All the tests related to this PR seem to be working fine in `master` now. I do not know why?

**Comment by hargup** [comment]
> All the tests related to this PR seem to be working fine in `master` now.

Use `git bisect` to figure out which commit fixed this and also add test for it.

**Pull request closed without merging**